internal/devserver/ts_devserver.bzl

```diff
@@ -47,6 +47,20 @@ def _ts_devserver(ctx):
   else:
     workspace_name = ctx.workspace_name
 
+  ctx.actions.write(ctx.outputs.concat_files_manifest, "".join([
+    # TODO: change to $(rootpath) after upgrading users to Bazel 0.8
```

Comment:
maybe we should do it now

Reply:
`expand_path_into_runfiles` is in `rules_nodejs` so are you think there would be something added to `rules_nodejs` to use here that deals with `$(rootpath)`?

Reply:
we shouldn't need to use any helper from rules_nodejs, `$(rootpath)` / `ctx.expand_location` should do exactly this